src/reader.rs

```diff
@@ -353,7 +353,10 @@ where
                         }
                         outer = outer_ring.remove(0);
                     }
-                    b"innerBoundaryIs" => inner = self.read_boundary(b"innerBoundaryIs")?,
+                    b"innerBoundaryIs" => {
+                        let mut boundary = self.read_boundary(b"innerBoundaryIs")?;
```

Comment:
Do we need the intermediate variable or would this be fine?

```rust
inner.append(&mut self.read_boundary(b"innerBoundaryIs")?);
```

Reply:
Great point, thanks! Just made the change.